gyp/pylib/gyp/generator/make.py

```diff
@@ -916,11 +916,12 @@ def WriteActions(self, actions, extra_sources, extra_outputs,
       # libraries, but until everything is made cross-compile safe, also use
       # target libraries.
       # TODO(piman): when everything is cross-compile safe, remove lib.target
-      self.WriteLn('cmd_%s = LD_LIBRARY_PATH=$(builddir)/lib.host:'
-                   '$(builddir)/lib.target:$$LD_LIBRARY_PATH; '
+      subst_builddir = '$(subst \',\'\\\'\',$(builddir))'
```

Comment:
nit: I'd rather you use the `"` string delimiter and the `r` prefix so you get:
```python
     subst_builddir = r"$(subst ', '\'', $(builddir))"
```
Also IMHO the arg to `self.WriteLn` should be constructed before the call
```
     action_cmd = "..." \
       "..." % (...)
     self.WriteLn(action_cmd)
```